set.mm

```diff
@@ -180023,6 +180111,30 @@ obviously not a bijection (by Cantor's theorem ~ canth2 ), and in fact
     ( cz wcel c1 cgcd co wceq 1z gcdcom mpan gcd1 eqtrd ) ABCZDAEFZADEFZDDBCMNO
     GHDAIJAKL $.
 
+  ${
+    gcdmuld.1 $e |- ( ph -> M e. NN0 ) $.
+    gcdmuld.2 $e |- ( ph -> N e. ZZ ) $.
+    $( The greatest common divisor of a nonnegative integer and a multiple of
+       it is itself.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    gcdmuld $p |- ( ph -> ( M gcd ( N x. M ) ) = M ) $=
+      ( cc0 cgcd co cmul caddc cz wcel wceq nn0zd 0zd gcdaddm syl3anc nn0gcdid0
+      cn0 syl zmulcld zcnd addid2d oveq2d 3eqtr3rd ) ABFGHZBFCBIHZJHZGHZBBUGGHA
+      CKLBKLFKLUFUIMEABDNZAOCBFPQABSLUFBMDBRTAUHUGBGAUGAUGACBEUJUAUBUCUDUE $.
+  $}
+
+  ${
+    gcddvdsd.1 $e |- ( ph -> M e. NN ) $.
+    gcddvdsd.2 $e |- ( ph -> N e. ZZ ) $.
+    gdcdvdsd.3 $e |- ( ph -> M || N ) $.
+    $( The greatest common divisor of a positive integer and another integer it
+       divides is itself.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    gcddvdsd $p |- ( ph -> ( M gcd N ) = M ) $=
```

Comment:
not deduction form of ~gcddvds. Maybe ~dvdsgcdidd (analogous to ~nndvdslegcd ).

Reply:
Renamed to ` dvdsgcdidd `.